src/resources/ModifierTemplates/ModifierTemplates.ts

```diff
@@ -0,0 +1,15 @@
+import Resource from '../Resource';
+import {OrganizationDefinitionModel} from '../OrganizationDefinitions/OrganizationDefinitionInterfaces';
+import {StatementDetailModel} from './ModifierTemplateInterfaces';
+
+export default class ModifierTemplates extends Resource {
```

Comment:
This new resource and the other one (OrganizationDefinitions) will have to be added either to the base class [`PlatformResources`](https://github.com/coveo/platform-client/blob/master/src/resources/PlatformResources.ts) or to another resource (example [here](https://github.com/coveo/platform-client/blob/master/src/resources/UsageAnalytics/UsageAnalytics.ts)) or else you won't be able to use them from the platform client instance like `platform.???.listPossibleStatements()`

Reply:
Modifier templates are global, they do not belong to an Organization; so they need their own ModifierTemplateResources?

Should I rename everything from "OrganizationDefinition" to "Definition"? It seems like everything is assumed to belong to an org.

Reply:
Tried something based on what already exists, tell me if it fits.